src/Compilers/CSharp/Portable/Errors/ErrorFacts.cs

```diff
@@ -573,6 +573,8 @@ internal static bool IsBuildOnlyDiagnostic(ErrorCode code)
                 case ErrorCode.ERR_EncUpdateFailedDelegateTypeChanged:
                 case ErrorCode.ERR_CannotBeConvertedToUtf8:
                 case ErrorCode.ERR_FileTypeNonUniquePath:
+                    // Update src\EditorFeatures\VisualBasic\LanguageServer\VisualBasicLspBuildOnlyDiagnostics.vb
```

Comment:
Another approach here would be to move this helper method to a separate file in a separate helper class and then link this file into both the projects? I'll let the compiler team decide their preference.

Reply:
I would prefer this approach.

Reply:
This approach doesn't work for us because it's all based on enums taht themselves are not shared with the IDE.  This space is effectively opaque to us.